### PR TITLE
Add a environment variable to configure max log characters for functions

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

It adds a new environment variable `_APP_FUNCTIONS_MAX_OUTPUT` which will configure the max length of logs stored by appwrite from function executions. The default has also been raised from 4000 to 65535 which is the limit we can store with a TEXT Type in our MariaDB Database since that's the type that is storing our logs.

## Related PRs and Issues
https://github.com/appwrite/appwrite/issues/1334

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes, I have.

### CODE REVIEW

File: app/config/variables.php [hunk 1]
```diff
@@ -444,6 +444,15 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_FUNCTIONS_OUTPUT_MAX',
+                'description' => 'The maximum length of characters a function execution is allowed to output. The default value is 65535',
+                'introduction' => '0.9.0',
+                'default' => '65535',
```
Comment (TorstenDittmann) [inline app/config/variables.php:451]: This is the maximum anyway right?

**Review by TorstenDittmann — APPROVED** [review]

File: app/config/variables.php (hunk 1 above)
Comment (PineappleIOnic) [reply app/config/variables.php:451]: Yeah it is the max currently using the TEXT Type in MariaDB. Not sure about the rewrite though.

Comment (TorstenDittmann) [reply app/config/variables.php:451]: Lets have this in the description - otherwise people might get confused when they raise this value 👍🏻

Comment (PineappleIOnic) [reply app/config/variables.php:451]: Done :+1:

**Comment by eldadfux** [comment]
Keeping this PR pending for the new DB changes which will allow more flexibility around the output length

**Pull request closed without merging**